virttest/backends/virttools/cfg/default_tests

```diff
@@ -0,0 +1 @@
+
```

Comment:
This file only has a blank line, is it necessary? (Can we remove this line? or this file?

Reply:
I successfully ran the test case without the file so I removed it :)